docs/delegation_design_spec/delegation_design_spec.tex

```diff
@@ -1205,12 +1205,14 @@ \subsubsection{Operational Key Certificates}
 They are included in the message (e.g.~block header) and the message
 itself is signed with \(sks_\text{hot}\).
 
-Operational keys will use a key evolving signature (KES) scheme.
+Operational keys will use key evolving signatures (KES). To be precise, we will
+use keys according to the MMM scheme \citep{cryptoeprint:2001:034}, with 1-day
+regular updates and a key lifetime of \(2^7 = 128\) days, a little over three
+months.
```

Comment:
We should perhaps clarify that this is the number of slots equivalent to one day. And I think the terminology is key "evloution" rather than "update".

Reply:
Thanks!